dockerfile/cuda11.1.1.dockerfile

```diff
@@ -2,7 +2,7 @@ FROM nvcr.io/nvidia/pytorch:20.12-py3
 
```

Comment:
Add comments for all version information.

Reply:
added